core/src/main/scala/kafka/cluster/Replica.scala

```diff
@@ -83,7 +83,6 @@ class Replica(val brokerId: Int, val topicPartition: TopicPartition) extends Log
     lastFetchLeaderLogEndOffset = leaderEndOffset
     lastFetchTimeMs = followerFetchTimeMs
     updateLastSentHighWatermark(lastSentHighwatermark)
-    trace(s"Updated state of replica to $this")
```

Comment:
I don't think this trace call is worthwhile given the cost. It would be hard to turn it on without being deluged with logs.